app.rb

```diff
@@ -0,0 +1,84 @@
+require_relative 'database_connection_setup'
```

Comment:
So you could also have popped a view helpers file in here to help define the user. it looks like you did have the current user as a method, but extracting into a 'helper file' might have helped with file/ logic structure of your app. 
<img width="683" alt="Screenshot 2020-06-08 at 10 33 46" src="https://user-images.githubusercontent.com/63285668/84016706-8ff9c500-a975-11ea-9905-f8dd28af9683.png">